build/generate-deploy-index/index.js

```diff
@@ -25,17 +26,31 @@ ensureDirectory(STYLEGUIDE_DIST);
 
 const STYLEGUIDE_DEPLOYS = getDirectories(STYLEGUIDE_DIST).filter((d) => d !== '0.0.0-semantic-release');
 const LAB_DEPLOYS = getDirectories(LAB_DIST);
+const additionalVersions = ['latest', 'latest-preview'];
 
 const VERSION_REGEX = /^\d+\.\d+\.\d+/;
+function isNumericVersion(deployName) {
+	return VERSION_REGEX.test(deployName);
```

Comment:
I think we should update the code to throw away `VERSION_REGEX` and `isNumericVersion` and use this instead:
```
const semverValid = require('semver/functions/valid');
```